fzf-git.sh

```diff
@@ -307,6 +307,23 @@ _fzf_git_worktrees() {
     " "$@" |
   awk '{print $1}'
 }
+
+_fzf_git_list_bindings(){
+  cat <<'EOF'
+
```

Comment:
for your information: the leading newline was a style choice to visually separate the help message from the user's prompt. can also be removed if u not approve of it.